internal/db/migration/000005_rename_to_singular.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE "models" RENAME TO "model";
```

Comment:
Besides changing the table name, we also need to change the constraint name such as
```
ALTER TABLE model RENAME CONSTRAINT models_pkey TO model_pkey
```